options/locale/locale_en-US.ini

```diff
@@ -308,6 +308,7 @@ repos = Repositories
 users = Users
 organizations = Organizations
 search = Search
+goto = Goto
```

Comment:
Would say make this `go_to = Go to`. "Goto" always makes me think of the same-named statement in some programming languages, so clear english seems better.